.github/workflows/appdistribution.yml

```diff
@@ -0,0 +1,52 @@
+name: appdistribution
+
+on:
+  pull_request:
+    paths:
+    - 'FirebaseAppDistribution**'
+    - '.github/workflows/appdistribution.yml'
+    - 'Gemfile'
+  schedule:
+    # Run every day at 3am (PST) - cron uses UTC times
+    # This is set to 3 hours after zip workflow finishes so zip testing can run after.
+    - cron:  '0 11 * * *'
+
+jobs:
+  pod-lib-lint:
+    # Don't run on private repo unless it is a PR.
+    if: github.repository != 'FirebasePrivate/firebase-ios-sdk' || github.event_name == 'pull_request'
+
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        target: [ios]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: Build and test
+      run: |
+       scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseAppDistribution.podspec \
+         --platforms=${{ matrix.target }} --ignore-local-podspecs=FirebaseAppDistribution.podspec
+
+  appdistribution-cron-only:
+    if: github.event_name == 'schedule' && github.repository != 'FirebasePrivate/firebase-ios-sdk'
+
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        target: [ios, tvos, macos]
+        flags: [
+          '--use-modular-headers',
+          '--use-libraries'
+        ]
+    needs: pod-lib-lint
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: PodLibLint App Distribution Cron
+      run: |
+       scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseAppDistribution.podspec \
+         --platforms=${{ matrix.target }} ${{ matrix.flags }} \
+         --ignore-local-podspecs=FirebaseAppDistribution.podspec
```

Comment:
The `--ignore-local-podspecs` should not be here.

Reply:
Done!